include/glow/Importer/Caffe2ModelLoader.h

```diff
@@ -51,14 +51,20 @@ class Caffe2ModelLoader
   llvm::Error loadWeightsFromNet(caffe2::NetDef &net);
 
   /// Loads an individual weight \p op.
+  /// If Bindings are provided creates Placeholders vs Constants.
   llvm::Error loadWeight(const caffe2::OperatorDef &op);
 
   /// Load the structure of the network from the 'net' file.
-  llvm::Error loadNetwork(caffe2::NetDef &net);
+  /// If Bindings are provided creates Placeholders vs Constants.
+  llvm::Error loadNetwork(caffe2::NetDef &net,
+                          PlaceholderBindings *bindings = nullptr);
 
   /// Load the operator \p op into the network. This creates one or more nodes
   /// in the network.
-  llvm::Error loadOperator(const caffe2::OperatorDef &op);
+  /// \p bindings is an optional parameters, if it is provided then training
```

Comment:
`bindings is an optional parameters. If it is provided, then trainable weights will be added to the graph as Placeholders.`